test/blindrsa.test.ts

```diff
@@ -78,6 +78,21 @@ async function keysFromVector(v: Vector, extractable: boolean): Promise<CryptoKe
     return { privateKey, publicKey };
 }
 
+test('Parameters', () => {
+    const hash = 'SHA-384';
+    const vectorList = [
+        { hash, saltLength: 0x30, suite: SUITES.SHA384.PSS.Deterministic() },
```

Comment:
probably need to merge before/after #7 and rename `SUITES` accordingly

Reply:
that will be solved in the other PR.